### DISCUSSION TITLE
上报群消息时包含群名称

### DISCUSSION THREAD

**Discussion by imyMuyang** [Mrs4s/go-cqhttp#1658] [Ideas]
如题，目前 OneBot v11 与 v12 标准上报的都没有包含群名称，一般来说没有上报这个的必要，但是如果要查询数据的话一直用API还是有点麻烦，可不可以直接上报的时候就顺带报过来，比如说在"group_id"字段后面加个"group_name"字段

**Comment by Akegarasu** [comment]
没必要，还增大上报数据，自己做个缓存就好
